src/Styles/Style.ts

```diff
@@ -0,0 +1,11 @@
+export const header1 = "text-2xl font-medium mb-1";
```

Comment:
Since we're allowed to use external libraries in this project, I would recommend you look at styled components once. That could help make this file better. I think

Reply:
Yes, I am using TailwindCSS for this project. Surely will checkout the other libraries as well 🙂